content/panorama/scripts/team_select.js

```diff
@@ -30,6 +30,7 @@ function hidePregameUI () {
 function showPregameUI () {
   FindDotaHudElement('PreGame').style.opacity = 1;
   FindDotaHudElement('PreGame').style.visibility = 'visible';
+  FindDotaHudElement('PreGame').FindChildTraverse( 'GameModeLabel' ).text = "Open Angel Arena";
```

Comment:
I just thought of that, because voting result was not outputted on pregame screen.
If you think that it should be done through localization, I'll revert it and create an issue, because I'm not familiar with that.